.vib/grafana/vib-platform-verify.json

```diff
@@ -23,7 +23,7 @@
           "target_platform": {
             "target_platform_id": "{TARGET_PLATFORM}",
             "size": {
-              "name": "S4"
+              "name": "L4"
```

Comment:
Grafana tests had a lot of random failures with pages not loading so I increased the target platform size. This helped.

Reply:
I agree increasing the cluster size is a viable option but be aware you jumped from `S4 -> L4`, a x4 increase in resources. Just to note that we are using an S4 for BTS testing and there may be an intermediate cluster size that could work for this case.

Reply:
Together with @joancafom we checked the BTS tests, they are using only endpoints, no UI interactions. 
The problem I had was that the UI elements weren't loading properly and the Cypress tests would fail. Increasing the cluster size helped with this.